src/drivers/drivers_options.cmake

```diff
@@ -0,0 +1,36 @@
+#********************************************************************
+#        _       _         _
+#  _ __ | |_  _ | |  __ _ | |__   ___
+# | '__|| __|(_)| | / _` || '_ \ / __|
+# | |   | |_  _ | || (_| || |_) |\__ \
+# |_|    \__|(_)|_| \__,_||_.__/ |___/
+#
+# www.rt-labs.com
+# Copyright 2018 rt-labs AB, Sweden.
+#
+# This software is dual-licensed under GPLv3 and a commercial
+# license. See the file LICENSE.md distributed with this software for
+# full license information.
+#*******************************************************************/
+message(STATUS "PNET_OPTION_DRIVER_ENABLE ${PNET_OPTION_DRIVER_ENABLE}")
+message(STATUS "Driver options configuration")
+
+macro(SUBDIRLIST result curdir)
+  FILE(GLOB children RELATIVE ${curdir} ${curdir}/*)
```

Comment:
We haven't used globbing before in this project. It is generally not recommended (although I have used it in other projects without issues). But see the note at https://cmake.org/cmake/help/latest/command/file.html#filesystem. If globbing is used, should we not also set CONFIGURE_DEPENDS here?

I assume the purpose is to be able to drop-in new drivers. If the driver is supplied as a patch then it would be fairly easy for the patch to also update a simple list of all available drivers.

Nitpick: This project has used lower-case for all cmake commands

Reply:
I also prefer a list of supported drivers.

Reply:
Previously I said that it should pick up all drivers in the directory, but maybe it is better to explicitly list the drivers.